collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/MovingBackwardsTest.kt

```diff
@@ -0,0 +1,69 @@
+package org.odk.collect.android.feature.settings
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain
+
+@RunWith(AndroidJUnit4::class)
+class MovingBackwardsTest {
```

Comment:
What's the thinking around adding this instead of adding tests to `FormEntryAccessPreferencesFragmentTest`?

Reply:
We can't test disabling moving backwards in isolation (using fragment-scenario like in `FormEntryAccessPreferencesFragmentTest`) because that fragment displays `MovingBackwardsDialog` which calls a listener from `ProjectPreferencesActivity` and then it calls a method from `FormEntryAccessPreferencesFragment`, so there is that activity involved.